src/scripting.rs

```diff
@@ -106,14 +106,14 @@ echo \
         );
         let combined = script_a + script_b;
 
-        let expected = r#"#!/bin/bash
```

Comment:
Just curious why you moved away from raw strings here?

Reply:
I was somewhat curious here as well when I looked back over it, but that was a `cargo fmt` thing as far as I can remember. This has been open for awhile now though and I can't remember 100% :smile: 

I'll add them back in in a second and see whether I can confirm that.

Reply:
I couldn't reproduce it, not sure how that happened :thinking: 

It seems it doesn't make a difference either way, but I've restored it so that it reduces the overall diff :smile: